src/main/java/com/deloitte/elrr/entity/AuditLog.java

```diff
@@ -0,0 +1,77 @@
+package com.deloitte.elrr.entity;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import org.hibernate.annotations.JdbcType;
+import org.hibernate.dialect.PostgreSQLEnumJdbcType;
+
+import com.deloitte.elrr.entity.types.ActionType;
+import com.deloitte.elrr.entity.types.SvcMethod;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "audit_log")
+@RequiredArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class AuditLog extends com.deloitte.elrr.entity.Entity {
+
+    @Column(name = "timestamp")
+    private LocalDateTime timestamp;
```

Comment:
We're changing everything to ZonedDateTime. Please follow suit here, should be drop-in replacement.

Reply:
https://github.com/US-ELRR/elrrserviceentities/pull/41/commits/361c096aed1fb66ece16ee4839a281d348e79964